src/components/MenuTab.tsx

```diff
@@ -0,0 +1,46 @@
+import clsx from "clsx";
+import type { ReactNode } from "react";
+import { useState } from "react"
+
+export type MenuItem = {
+  title: string
+  content: ReactNode
+  onChange?: (index: number) => void
+}
+
+export const MenuTab = ({ items, lightBackground, isSelected }: { items: Array<MenuItem>, lightBackground?: boolean, isSelected?: number }) => {
+  const [ currentIdx, setCurrentIdx ] = useState(0)
+  const equalSplitValue = 100 / items.length + "%"
+  const selectedIdx = isSelected ? isSelected : currentIdx
+  return (
+    <>
+      <div className="flex flex-row justify-center overflow-hidden">
+        { items.map((item, idx) => (
+          <>
+            <div
+              key={ item.title + idx + "--tab" }
+              className={clsx("btn rounded-none", {
+                        "btn--outlined--light": !lightBackground,
+                        "btn--outlined": lightBackground,
+                        "opacity-50": selectedIdx == idx,
+                        "underline": selectedIdx == idx
+                    })}
+              style={{ width: equalSplitValue }}
```

Comment:
I think you can add the width with tailwind as well, like so:

```suggestion
              className={clsx("btn rounded-none", `w-[${equalSplitValue}]` ,{
                        "btn--outlined--light": !lightBackground,
                        "btn--outlined": lightBackground,
                        "opacity-50": selectedIdx == idx,
                        "underline": selectedIdx == idx
                    })}
```

Reply:
It does not work as you suggested but I fixed it using another CSS trick. Previously, I calculated it with styles because `flex-1` caused some problems with the inner tabs in my fork but it is a better solution and hopefully we can work it out when I encounter the problem again.